tests/ft_erlang.vader

```diff
@@ -30,32 +30,29 @@ Execute (erlc: output to _build/neomake if _build is present):
     execute 'cd ' . old_cwd
   endtry
 
-Execute (erlc: rebar3 default profile - no rebar.config):
+Execute (erlang#EbinDirs: rebar3 default profile - no rebar.config):
   let old_cwd = getcwd()
   try
-    let root = tempname()
+    let root = tempname() . '/'
     call mkdir(root, 'p')
```

Comment:
Seems like vim73 does not like the extra slash here with the mkdir.

Reply:
Please also use `neomake#utils#Slash()` for Windows compatibility in the future.. (related: see utils in 119dc9be)